code/framework/src/services/masterlist.cpp

```diff
@@ -54,18 +117,18 @@ namespace Framework::Services {
                     {"current_players", std::to_string(_storedInfo.currentPlayers)},
                 };
 
-                // Send the request
-                // Make sure we can only handle valid responses
-                const auto res = _client->Post("/rcon/ping", params);
+                // Send the request to the masterlist
+                const auto res = masterlistClient->Post("/ping", params);
                 if (!res) {
                     Logging::GetLogger(FRAMEWORK_INNER_SERVER)->error("Failed to ping masterlist server: {}", res.error());
                 }
-                if (res->status != 200 && res->status != 201) {
+                else if (res->status != 200 && res->status != 201) {
```

Comment:
might be a good idea to also check if we got 403 from masterlist (say when jwt is expired or forcibly removed, so server doesn't wait for re-auth and immediately calls Authenticate (with cooldown ofc so we don't spam))